src/branches/Leaf.ts

```diff
@@ -54,9 +55,20 @@ export default class Leaf extends Container {
 		this.addChild(this.amountText);
 	}
 
-	private getTint(amount: number): [number, number | undefined] {
-		// TODO: Check date and return correct, color
-		return [0x8eb332, undefined];
+	private getTint(amount: number): [number, number | undefined, number] {
+		// Graduation sakura colors
+		if(Date.now() > new Date(2025, 0, 3).getTime()) {
+			const sakuraColors: [number, number | undefined, number][] = [
+				[0xFF8BD4, 0x000000, 1.75],
+				[0xFEB5E3, 0x000000, 1.75],
+				[0xFCBCDF, 0x000000, 1.75]
+			];
+
+			const randNum = getRandomNumber(0, sakuraColors.length);
+			return sakuraColors[randNum];
+		}
+
+		return [0x8eb332, undefined, 1];
```

Comment:
Natural green still needs to be implemented

Reply:
Implemented